### PR TITLE
feat: support `PLAYWRIGHT_GLOBAL_INSTALL=1` env variable

### LINKED ISSUE #1102: [Feature] support global installation of browsers
I'm currently working on different projects where I play around with playwright. Downloading the latest browser versions for each single project is slowing me down and also uses a lot of disk-space.

~~Due to https://github.com/microsoft/playwright/issues/1085 I also need to re-download the browsers over and over again whenever I update my node_modules.~~ (Update: is fixed)

I like [the way Cypress manages binary installs](https://docs.cypress.io/guides/getting-started/installing-cypress.html#Advanced) by allowing configuration through environment variables.

I'd be happy to investigate further and propose an API/implementation if you think this is a good idea.

### PR DESCRIPTION
**Pull request by aslushnikov**
You can install playwright with

```
PLAYWRIGHT_GLOBAL_INSTALL=1 npm i playwright
```

to make it use a single shared location for all browser
downloads.

Fixes #1102

### CODE REVIEW

File: download-browser.js [hunk 1]
```diff
@@ -14,10 +14,16 @@
...
+const path = require('path');
 const browserFetcher = require('./lib/server/browserFetcher.js');
 const packageJSON = require('./package.json');
+const envPaths = require('env-paths');
+const appPaths = envPaths('playwright');
 
-async function downloadBrowserWithProgressBar(downloadPath, browser, version = '') {
+async function downloadBrowserWithProgressBar(downloadPath, browser, respectGlobalInstall = false) {
+  const PLAYWRIGHT_GLOBAL_INSTALL = respectGlobalInstall ? getFromENV('PLAYWRIGHT_GLOBAL_INSTALL') : false;
+  if (!!PLAYWRIGHT_GLOBAL_INSTALL && PLAYWRIGHT_GLOBAL_INSTALL.toLowerCase().trim() !== 'false')
```
Comment (dgozman) [inline download-browser.js]: nit: maybe respect 0 similar to false?

File: download-browser.js [hunk 2]
```diff
@@ -14,10 +14,16 @@
  * limitations under the License.
  */
 const fs = require('fs');
+const path = require('path');
 const browserFetcher = require('./lib/server/browserFetcher.js');
 const packageJSON = require('./package.json');
+const envPaths = require('env-paths');
+const appPaths = envPaths('playwright');
```
Comment (dgozman) [inline download-browser.js]: nit: I'd inline to the callsite.

**Review by dgozman — APPROVED** [review]

File: download-browser.js (hunk 2 above)
Comment (aslushnikov) [reply download-browser.js]: done

File: download-browser.js (hunk 1 above)
Comment (aslushnikov) [reply download-browser.js]: done

**Review by awibox — APPROVED** [review]

**Pull request merged**